src/dialogs/mqtt-device-debug-info-dialog/dialog-mqtt-device-debug-info.ts

```diff
@@ -0,0 +1,208 @@
+import {
+  LitElement,
+  css,
+  html,
+  CSSResult,
+  TemplateResult,
+  customElement,
+  property,
+} from "lit-element";
+import "../../components/ha-dialog";
+import "../../components/ha-switch";
+import { computeDeviceName } from "../../data/device_registry";
+import { computeStateName } from "../../common/entity/compute_state_name";
+import { haStyleDialog } from "../../resources/styles";
+// tslint:disable-next-line: no-duplicate-imports
```

Comment:
Remove this and use `import type` to import type definitions.

Reply:
Fixed